docassemble/AssemblyLine/language.py

```diff
@@ -13,6 +14,32 @@
 ]
 
 
+def _package_name(package_name: Optional[str] = None):
+    """Get package name without the name of the current module"""
+    if not package_name:
+        package_name = __name__
+    try:
+        return ".".join(package_name.split(".")[:-1])
+    except:
+        return package_name
+
+
+def get_local_languages_yaml():
+    current_package_name = _package_name()
+
+    this_yaml = path_and_mimetype(f"{current_package_name}:data/sources/languages.yml")[
+        0
+    ]
+    try:
+        local_yaml = path_and_mimetype("data/sources/languages.yml")[0]
+    except:
+        local_yaml = None
+
+    if local_yaml and os.path.isfile(local_yaml):
+        return local_yaml
+    return this_yaml
```

Comment:
```suggestion
    try:
        local_yaml = path_and_mimetype("data/sources/languages.yml")[0]
    except:
        local_yaml = None

    if local_yaml and os.path.isfile(local_yaml):
        return local_yaml
        
    al_package_name = _package_name()
    this_yaml = path_and_mimetype(f"{al_package_name}:data/sources/languages.yml")[
        0
    ]
    return this_yaml
```

Just moved `this_yaml` to the bottom (no need to create it if it's not going to be used), and renamed `current` to `al`, which makes things clearer IMO.